Pod/Core/XCTestCase+Gherkin.swift

```diff
@@ -189,6 +189,13 @@ public extension XCTestCase {
         self.performStep(expression + " \(DataTable(values()))", keyword: "Given", file: file, line: line)
     }
 
+    /**
+     Run the step matching the specified expression followed by a data table as a raw string
+     */
+    func Given(_ expression: String, file: String = #file, line: Int = #line, dataTable values: () -> String) {
+        self.performStep(expression + " \(values().components(separatedBy: "\n").joined(separator: ","))", keyword: "Given", file: file, line: line)
```

Comment:
This bit of formatting is repeated in all new functions. It would be nice to have this formatting done in a single place, e.g.

```suggestion
        self.performStep(expression + " \(formatNativeDataTable(rawString: values())", keyword: "Given", file: file, line: line)
```

Reply:
good suggestion